frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -226,25 +228,41 @@ private void refreshModel(
 
     setModel(new GraphTableModel(repositoryGraph));
 
-    Set<String> skippedBranchNames = repositorySnapshot.getSkippedBranchNames();
-    if (skippedBranchNames.nonEmpty()) {
-      val notification = getSkippedBranchesNotification(repositorySnapshot, gitRepository);
-      VcsNotifier.getInstance(project).notify(notification);
-    }
+    if (!macheteFileIsOpened(macheteFilePath)) {
+      // notify if branch listed in machete file does not exist
+      Set<String> skippedBranchNames = repositorySnapshot.getSkippedBranchNames();
+      if (skippedBranchNames.nonEmpty()) {
+        val notification = getSkippedBranchesNotification(repositorySnapshot, gitRepository);
+        VcsNotifier.getInstance(project).notify(notification);
+      }
 
-    Set<String> duplicatedBranchNames = repositorySnapshot.getDuplicatedBranchNames();
-    if (duplicatedBranchNames.nonEmpty()) {
-      // This warning notification will not cover other error notifications (e.g. when rebase errors occur)
-      VcsNotifier.getInstance(project).notifyWarning(/* displayId */ null,
-          getString("string.GitMachete.EnhancedGraphTable.duplicated-branches-text"),
-          String.join(", ", duplicatedBranchNames));
+      // notify if branch name listed in machete file appears more than once
+      Set<String> duplicatedBranchNames = repositorySnapshot.getDuplicatedBranchNames();
+      if (duplicatedBranchNames.nonEmpty()) {
+        // This warning notification will not cover other error notifications (e.g. when rebase errors occur)
+        VcsNotifier.getInstance(project).notifyWarning(/* displayId */ null,
+            getString("string.GitMachete.EnhancedGraphTable.duplicated-branches-text"),
+            String.join(", ", duplicatedBranchNames));
+      }
     }
 
     repaint();
     revalidate();
     doOnUIThreadWhenReady.run();
   }
 
+  @UIEffect
+  private boolean macheteFileIsOpened(Path macheteFilePath) {
+    val fileEditorManager = FileEditorManager.getInstance(project);
+    val macheteVirtualFile = List.of(fileEditorManager.getOpenFiles())
```

Comment:
Out of curiosity... why `getOpenFiles()` returns files not a file? I assume that it does not include files that you can access from the tabs but it does include files opened in the split editor window in particular, right?

What I mean by tabs:
<img width="567" alt="image" src="https://user-images.githubusercontent.com/19799111/197196890-96638d34-eec7-45e8-8dbe-1c4bb6fc7589.png">

What I mean by split editor window:
![image](https://user-images.githubusercontent.com/19799111/197197346-9a72eeaf-23ac-4371-97f7-12a1b10f8454.png)

Reply:
`getOpenFiles()` returns all opened files. It returns all files that are opened in the editor tabs.

Reply:
hm, so it's not what we want , right? the notification should be skipped (/disabled) only when the machete file is visible, right?

Reply:
I am not sure. It seemed to me that notifications should not be displayed if the file is opened. @PawelLipski please specify :)

Reply:
> only when the machete file is visible, right?

Right, +1, my fault. Of course I meant **opened & focused** aka visible, not just opened but unfocused. Lemme fix that on the issue. Sorry for the confusion, pls rework @LukaszKontowski

Reply:
fixed :)